persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/DsePersistence.java

```diff
@@ -531,4 +587,31 @@ private Object queryOrId(Statement statement) {
       }
     }
   }
+
+  private static class PullRequestGetter {
+    private static final Field schedulerField;
+    private static final Method nonCompletedPullRequestsMethod;
+    private static final Object scheduler; // PullRequestScheduler
+
+    static {
+      try {
+        schedulerField = MigrationManager.class.getDeclaredField("scheduler");
+        schedulerField.setAccessible(true);
+        scheduler = schedulerField.get(MigrationManager.instance);
```

Comment:
why not cast to `(MigrationManager`) and change the `scheduler` type from `Object` to `MigrationManager`?

Reply:
`scheduler` is of type `PullRequestScheduler`, which is not `public`